docs/tracing/10-01-traces-troubleshooting.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Basic Troubleshooting
+type: Troubleshooting
+---
+
+## Jaeger shows only a few traces
+
+The current istio-pilot settings define the trace sampling rate at `1.0`, where `100` is the maximum. This means that only 1 out of 100 requests is sent to Jaeger for trace recording. To change this system behavior, run:
```

Comment:
Istio Pilot, not "istio-pilot". 
https://istio.io/docs/concepts/traffic-management/